components/camel-platform-http-vertx/pom.xml

```diff
@@ -47,6 +47,10 @@
             <artifactId>vertx-web</artifactId>
             <version>${vertx-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-http</artifactId>
```

Comment:
If only used for testing then set it to test scope

Reply:
done